### PR TITLE
Use probe data from all buildhub revisions instead of release tags

### LINKED ISSUE #225: P-S not properly parsing difference in `record_in_processes`
See https://github.com/mozilla/mozilla-schema-generator/issues/147

[Here](https://phabricator.services.mozilla.com/D78033) is where this field was added, and [here](https://phabricator.services.mozilla.com/D89401) is where it changed. The probe-info-service should have an entry for each of those, but instead it only has an entry for the latter.

### PR DESCRIPTION
**Pull request by Ben-Wu**
(fixes #225)

~~Seems to work from some quick tests so putting a draft up first.  I'll add unit tests and test more before marking as ready for review.~~ 👍

This is the output for the probe in question: https://gist.github.com/Ben-Wu/76f8eacf2da9afff7e2d9e80ea7ad3fb

### CODE REVIEW

**Comment by Ben-Wu** [comment]
Additional caveat is that the buildhub revisions are per push so the revisions it has are for the last change at the time of each push instead of every single change.  So if there are two patches in a single push that affects a probe, it will be reflected as one change.  This shouldn't matter at all, just some additional context to reason about things.

File: probe_scraper/scrapers/moz_central_scraper.py [hunk 1]
```diff
@@ -272,7 +276,7 @@ def scrape(folder=None, min_fx_version=None, max_fx_version=None, channels=None)
         folder = tempfile.mkdtemp()
 
     error_cache = load_error_cache(folder)
-    requests_cache.install_cache('probe_scraper_cache')
+    requests_cache.install_cache("probe_scraper_cache")
     results = defaultdict(dict)
 
     if channels is None:
```
Comment (fbertsch) [inline probe_scraper/scrapers/moz_central_scraper.py:282]: Looks to me like this is still scraping all the channels. We should explicitly only scrape release.

**Review by fbertsch — APPROVED** [review]
Nice, this was much simpler than I anticipated. r+wc

File: probe_scraper/scrapers/moz_central_scraper.py (hunk 1 above)
Comment (fbertsch) [reply probe_scraper/scrapers/moz_central_scraper.py:282]: My mistake, keep as-is

**Comment by Ben-Wu** [comment]
Created https://github.com/mozilla/probe-scraper/issues/230 for using revisions from buildhub

**Pull request merged**